recipes/libibverbs-cos6-x86_64/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: libibverbs-cos6-x86_64
+  version: 1.1.8
+
+source:
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/libibverbs-1.1.8-4.el6.x86_64.rpm
+    sha256: 706e5770d5c3e6c06f7c47a88936edbee56a003f31186ea4dd7e1de8b87903ad
+    folder: binary
+  - url: http://vault.centos.org/6.10/os/Source/SPackages/libibverbs-1.1.8-4.el6.src.rpm
+    folder: source
+
+build:
+  number: 0
+  noarch: generic
+  missing_dso_whitelist:
+    - '*'
+
+
+###########################################################
+# Commented out to avoid a conda-build bug.               #
+# xref: https://github.com/conda/conda-build/issues/3553  #
+###########################################################
+#
+#test:
+#  commands:
+#    - test -d "${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot"
```

Comment:
Currently adding a `test` section results in issue ( https://github.com/conda/conda-build/issues/3553 ). So it is commented for now.

Reply:
Have since added a workaround to add the tests back.